middleware/l10n.js

```diff
@@ -17,6 +17,9 @@ function appSupportsLocale(locale) {
 }
 
 export default ({ app, route, redirect, req }) => {
+  // Exit early if this is an auth callback
+  if (app.$auth && (route.path === app.$auth.options.redirect.callback)) return;
```

Comment:
Adapt this to also exit early if the route path is `app.$auth.options.redirect.login`